h2o-genmodel-extensions/xgboost/src/main/java/hex/genmodel/algos/xgboost/AuxNodeWeightsHelper.java

```diff
@@ -0,0 +1,60 @@
+package hex.genmodel.algos.xgboost;
+
+import biz.k11i.xgboost.tree.RegTree;
+import biz.k11i.xgboost.tree.RegTreeNodeStat;
+
+import java.lang.reflect.Field;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+public class AuxNodeWeightsHelper {
+
+    public static byte[] toBytes(double[][] auxNodeWeights) {
+        int elements = 0;
+        for (double[] weights : auxNodeWeights)
+            elements += weights.length;
+        int len = (1 + auxNodeWeights.length) * 4 + elements * 8;
```

Comment:
I would use: Integer.BYTES -> 4 and Double.BYTES -> 8
or if we use Java 7 then only comment what the number means

Reply:
yeah, genmodel is still Java 7 only - but will improve

Reply:
done